Applications/Games/GOG Galaxy/Online/script.js

```diff
@@ -0,0 +1,37 @@
+include(["engines", "wine", "quick_script", "online_installer_script"]);
+include(["engines", "wine", "plugins", "windows_version"]);
+include(["engines", "wine", "verbs", "corefonts"]);
+include(["engines", "wine", "verbs", "vcrun2017"]);
+include(["engines", "wine", "verbs", "xact"]);
+
+var installerImplementation = {
+    run: function () {
+        new OnlineInstallerScript()
+            .name("GOG Galaxy")
+            .category("Games")
+            .editor("GOG.com")
+            .applicationHomepage("https://www.gog.com/")
+            .author("Markus Ebner")
+            .url("https://content-system.gog.com/open_link/download?path=/open/galaxy/client/setup_galaxy_1.2.51.30.exe")
+            .checksum("faa29721de18a171561a3e4dfa243953ff98aa09")
+            .executable("GalaxyClient.exe")
+            .wineArchitecture("amd64")
+            .wineVersion(LATEST_STAGING_VERSION)
+            .wineDistribution("staging")
+            .preInstall(function (wine/*, wizard*/) {
+                wine.windowsVersion("win10");
```

Comment:
Unless it's needed to run the application, do not use it. Wine defaults to windows 7, and it should work for most apps.